modules/test/playwright/tests/frontend-data-set-views-web/tests/data-set-admin/creationActions.spec.ts

```diff
@@ -0,0 +1,152 @@
+/**
```

Comment:
We need a SF rule to replace the `(c) 2000` by `(c) 2024` like we have in java/jsp :smile: